molo/surveys/tests/test_templatetags.py

```diff
@@ -137,6 +144,16 @@ def test_get_survey_list_only_direct(self):
         self.assertTrue(
             self.translated_linked_survey not in context['surveys'])
 
+    def test_get_survey_list_only_yourwords(self):
+        context = Context({
+            'locale_code': 'en',
+            'request': self.request,
+        })
+        context = get_survey_list(context, only_yourwords=True)
+        self.assertTrue(len(context['surveys']) == 1)
```

Comment:
Could we please use `self.assertEqual(len(context['surveys']), 1)` instead? (Here and throughout this file).
It gives us more sensible error output when the test fails.